src/main.rs

```diff
@@ -122,16 +121,31 @@ fn find_binary_operators(
     }
     seq!(idx in 0..100 {
         if let Some(&op) = BINARY_OPERATORS.get(idx) {
+            let op_idx: OpIndex = OpIndex::new(idx + UNARY_OPERATORS.len());
```

Comment:
no need to specify the type here, especially we are using `OpIndex::new` so the type is very clear
```suggestion
            let op_idx = OpIndex::new(idx + UNARY_OPERATORS.len());
```